packages/components/src/templates/next/components/complex/Hero/Hero.tsx

```diff
@@ -29,15 +32,23 @@ const Hero = ({
             </div>
             {buttonLabel && buttonUrl && (
               <div className="flex flex-col justify-start gap-x-5 gap-y-4 sm:flex-row">
-                <LinkButton href={buttonUrl} size="lg">
+                <LinkButton
+                  href={getReferenceLinkHref(buttonUrl, site.siteMap)}
```

Comment:
If we inject the whole sitemap into components, would it be too heavy esp for sites like MOH?

Reply:
Checked with the latest build, there isn't any observable impact on the build times since this is mostly just passing the reference around.